glass/__init__.py

```diff
@@ -63,11 +63,10 @@
 ]
 
 
-import contextlib
-from importlib.metadata import PackageNotFoundError
-
-with contextlib.suppress(PackageNotFoundError):
-    from ._version import __version__, __version_tuple__
+try:  # noqa: SIM105
```

Comment:
Still don't know I feel about this https://docs.astral.sh/ruff/rules/suppressible-exception/

Reply:
:disappointed: `contextlib.suppress()` expresses intent clearer than the idiomatic way to suppress an exception? flake8-simplify need to have a long, hard look at themselves

Reply:
Generally speaking, it seems like ruff has a catalogue of rules from every niche style checker ever written, and I'm not sure a lot of them are good ideas...

Reply:
I think we should merge this. But ruff has a very active community and unpopular rules have been deprecated. For example, [ANN101](https://docs.astral.sh/ruff/rules/missing-type-self/#missing-type-self-ann101) was removed relatively quickly.